app/jobs/ingest_yaml_job.rb

```diff
@@ -107,6 +108,10 @@ def ingest_files(parent: nil, resource: nil, files: [])
       attach_files_to_work(resource, @file_sets) if @file_sets.any?
     end
 
+    def ingest_ocr(actor, f)
+      actor.create_content(ocr_file(f), "extracted_text") if ocr_file(f)
```

Comment:
'if ocr_file(f)' creates an open File that is never used, consuming a file handle that won't be released until the File is gc'ed.  Assign ocr_file(f) to a variable and test that instead?  If not passed to actor.create_content, it should be closed.  Conceivably a large batch load cound run out of file handles before the gc has a chance to finalize enough Files.